pyw3d/__init__.py

```diff
@@ -74,7 +74,7 @@ def __init__(self, message):
 
 from .features import W3DFeature
 from .project import W3DProject
-from .objects import W3DObject, W3DLink, W3DContent, W3DText, W3DImage, \
+from .objects import W3DObject, W3DLink, W3DShape, W3DContent, W3DText, W3DImage, \
```

Comment:
Watch out for the [79 character limit](https://www.python.org/dev/peps/pep-0008/#maximum-line-length). If you happen to think of it, it's a good idea to run your code through the pep8 program before committing, or better yet use an integrated style checker with your editor.